Marlin/src/MarlinCore.cpp

```diff
@@ -982,7 +982,7 @@ void setup() {
     serial_connect_timeout = millis() + 1000UL;
     while (!MYSERIAL1 && PENDING(millis(), serial_connect_timeout)) { /*nada*/ }
   #endif
-  SERIAL_ECHO_MSG("start");
+  SERIAL_ECHOLN("start");
```

Comment:
This used to use `SERIAL_ECHOLNPGM`. Is this going to fail on AVR?

Reply:
I tested on avr, and retested just now for piece of mind
Enabled both and got the expect results.
21:42:47.658 > echo:start
21:42:47.658 > start
21:42:47.658 >  External Reset
....

Reply:
But SERIAL_ECHOLNPGM is cleaner.